hardware/1Wire/1WireByKernel.cpp

```diff
@@ -551,7 +544,8 @@ void C1WireByKernel::GetDevice(const std::string& deviceName, /*out*/_t1WireDevi
 	device.family = ToFamily(deviceName.substr(0, 2));
 
 	// Device Id (6 chars after '.')
-	std_to_upper(deviceName.substr(3, 3 + 6 * 2), device.devid);
+	auto substr = deviceName.substr(3, 3 + 6 * 2);
+	std::transform(substr.begin(), substr.end(), device.devid.begin(), ::toupper);
```

Comment:
In our helper.h/cpp we have a function 'stdupper'

Reply:
Funny. The last commit converts it to be identical.